### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by Gabrielr2508**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----
This pull request is an on-going effort to provide the Portuguese translation.
-----
The following files are already translated or in progress:
- [x] _data/locales/pt.yml
- [x] _articles/pt/building-community.md
- [x] _articles/pt/best-practices.md
- [x] _articles/pt/code-of-conduct.md
- [x] _articles/pt/finding-users.md
- [x] _articles/pt/getting-paid.md
- [x] _articles/pt/how-to-contribute.md
- [x] _articles/pt/leadership-and-governance.md
- [x] _articles/pt/legal.md
- [x] _articles/pt/metrics.md
- [x] _articles/pt/starting-a-project.md

The following files are already reviewed or in progress:
- [x] _data/locales/pt.yml
- [x] _articles/pt/building-community.md
- [x] _articles/pt/best-practices.md
- [x] _articles/pt/code-of-conduct.md
- [x] _articles/pt/finding-users.md
- [x] _articles/pt/getting-paid.md
- [x] _articles/pt/how-to-contribute.md
- [x] _articles/pt/leadership-and-governance.md
- [x] _articles/pt/legal.md
- [x] _articles/pt/metrics.md
- [x] _articles/pt/starting-a-project.md

### CODE REVIEW

**Comment by Iosunlock** [comment]
Thx u infomation all eequested

**Comment by MikeMcQuaid** [comment]
@Gabrielr2508 Shout when it's ready to merge.

**Comment by k120563295** [comment]
我知道在這期間內有好友伸援手，但是對於網路不熟，全靠自己摸索就如同叫一個不認識字的人去環遊世界般的困難

**Comment by Gabrielr2508** [comment]
@k120563295 Sorry, but i can't understand chinese.

**Comment by k120563295** [comment]
您可以用 google 翻譯，造成困擾， sorry

Gabriel Rafael Gomes <notifications@github.com> 於 2018年10月2日 週二 19:01 寫道：

**Comment by igorsteinmacher** [comment]
@Gabrielr2508 shout if you need help on that. I'd be glad to help

**Comment by Gabrielr2508** [comment]
Hello, @igorsteinmacher. All help is welcome! 

The translation of these files has already begun or ended
_data/locales/pt.yml
_articles/pt/building-community.md
_articles/pt/best-practices.md

**Comment by jaymifoster** [comment]
google4

**Comment by Gabrielr2508** [comment]
> @Gabrielr2508 #779 should make it easier to translate headings.

@leolleocomp take a look.

**Comment by Gabrielr2508** [comment]
Sorry, i've clicked on wrong button. :S

**Comment by leolleocomp** [comment]
> > @Gabrielr2508 #779 should make it easier to translate headings.
> 
> @leolleocomp take a look.

Got it

**Comment by Gabrielr2508** [comment]
Hi, @MikeMcQuaid.

Could you clarify a doubt? We are actually translating the project into Brazilian Portuguese. Should we make this distinction between pt_br and pt_pt? Or do we consider all Portuguese?

Thank you.

**Comment by Gabrielr2508** [comment]
> Hi, @MikeMcQuaid.
> 
> Could you clarify a doubt? We are actually translating the project into Brazilian Portuguese. Should we make this distinction between pt_br and pt_pt? Or do we consider all Portuguese?
> 
> Thank you.

@MikeMcQuaid, could you answer please?

Thank you!

**Comment by MikeMcQuaid** [comment]
@Gabrielr2508 It's up to you, do whatever you think is best and set the locale/language accordingly.

**Comment by Gabrielr2508** [comment]
Hey guys, we finished the translation of the articles! Yaay! 😄 
We will start the review now and we will finalize soon.

**Comment by Gabrielr2508** [comment]
Hey, guys. Now it's done. I would like thank you very much to @leolleocomp and @krlsdu for the help.

😄 🎉

**Comment by Gabrielr2508** [comment]
Anyone can help us with the travis error?

**Comment by Gabrielr2508** [comment]
> Anyone can help us with the travis error?

Working on it

**Comment by Gabrielr2508** [comment]
> > Anyone can help us with the travis error?
> 
> Working on it

Done!

File: Gemfile.lock [hunk 1]
```diff
@@ -24,6 +24,7 @@ GEM
     ethon (0.12.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```
Comment (mlinksva) [inline Gemfile.lock]: I suspect a11ed7c1b942f2a262ce74cf402637503d15a13d should be reverted. I'm not sure why a translation would include node module updates. Am I missing something?

Comment (Gabrielr2508) [reply Gemfile.lock]: It was a mistake. Sorry.

I think now it's ok.

Thank you.

**Pull request merged**